india_banking/overrides/payment_order.py

```
@@ -6,29 +6,59 @@
+							link
+						)
 					),
 				)
+		if settings.enable_bank_balance_validation:
+			self.validate_bank_balance()
 		self.validate_bank_payment_request()
 
+	def validate_bank_balance(self):
+		try:
+			balance = get_bank_balance(self.company_bank_account)
+		except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)